src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_mysql.py

```diff
@@ -611,7 +611,7 @@ def _create_mysql_connection_strings(host, user, password, database):
                        "spring.datasource.password={password}",
         'node.js': "var conn = mysql.createConnection({{host: '{host}', user: '{user}', "
                    "password: {password}, database: {database}, port: 3306}});",
-        'php': "host={host} port=3306 dbname={database} user={user} password={password}",
+        'php': "$con=mysqli_init(); [mysqli_ssl_set($con, NULL, NULL, {{ca-cert filename}}, NULL, NULL);] mysqli_real_connect($con, '{host}', '{user}@{server}', '{password}', '{database}', 3306);",
```

Comment:
For flexible server, I think the user name has to be {user} only. For single server in the other file, {user}@{server} is correct.

Reply:
Good catch! Thank you for it!

Reply:
Thanks for the fix!